src/juju/index.js

```diff
@@ -80,12 +87,42 @@ async function loginWithBakery(visitPage, macaroonStore) {
   }
   const juju = await jujulib.connect(
     controllerURL,
-    generateConnectionOptions()
+    generateConnectionOptions(true)
   );
   const conn = await juju.login({});
+  // Ping to keep the connection alive.
+  conn.facades.pinger.pingForever(20000, err => {
+    if (err) {
+      console.log("unable to ping:", err);
```

Comment:
Should this be a console.error so we get reports from sentry?

Reply:
We don't have sentry yet but yep I can change that.

```suggestion
      console.error("unable to ping:", err);
```